core/pom.xml

```diff
@@ -14,6 +14,10 @@
     <name>TiSpark Project Core Internal</name>
     <url>http://github.copm/pingcap/tispark</url>
 
+    <properties>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+    </properties>
```

Comment:
That isn't very clear to me. If I don't set the JDK version, It will use the 1.6 language level, which will make us unable to use lambda, steam, and so on in the `core` module.